src/Illuminate/Database/Eloquent/Builder.php

```diff
@@ -822,10 +822,15 @@ protected function addUpdatedAtColumn(array $values)
             return $values;
         }
 
-        return Arr::add(
-            $values, $this->model->getUpdatedAtColumn(),
-            $this->model->freshTimestampString()
-        );
+        if (count($this->getQuery()->joins) > 0) {
```

Comment:
Like this is the PHP 7.2 break in case `joins` is e.g. null?

Reply:
@mfn Yep. Fixed in another PR

Reply:
@mfn https://github.com/laravel/framework/pull/26022